app/stac_api/templates/js/admin/asset_help_search.js

```diff
@@ -0,0 +1,72 @@
+window.onload = function() {
+    var innerhtml = `
+    Search Usage:
+    <ul>
+        <li>
+            [arg] will make a non exact search checking if [arg] is
+            part of the collection ID, the item ID and/or the asset ID
+        </li>
+        <li>
+            Multiple [arg] can be used, separated by spaces. This will search for all assets ID,
+            items ID / or collections ID containing all arguments.
+        </li>
+        <li>
+            ["arg"] will make an exact search on the asset ID field.
+        </li>
+        <li>
+          ["collectionID/itemID/assetID"] will make an exact search on a whole path.
+        </li>
+    </ul>
```

Comment:
Would be nice if you could reuse here the text from `SEARCH_TEXT_HELP_ITEM` somehow so it's just defined in one place

Reply:
My "problem" is that all texts are slightly different to account for small variations in how the search work depending on what you can search for. That being said, I think it would be possible to reuse the collections and items text. I'll look into it (the asset text is used only in the general asset page)